recipes/easy_profiler/all/conanfile.py

```diff
@@ -0,0 +1,87 @@
+from conans import ConanFile, tools, CMake
+import os
+import glob
+
+
+class EasyProfilerConan(ConanFile):
+    name = "easy_profiler"
+    description = "Lightweight profiler library for c++"
+    license = "MIT"
+    topics = ("conan", "easy_profiler")
+    homepage = "https://github.com/yse/easy_profiler/"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": True,
+        "fPIC": True
+    }
+    short_paths = True
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        # The windows build seems to be giving problems due to certain symbols
+        # not being exported properly for static libraries.
+        if self.options.shared is False and self.settings.os == "Windows":
```

Comment:
```suggestion
        if not self.options.shared and self.settings.os == "Windows":
```

The `is` comparator is a bit tricky in Python, we need to revisit it in Conan 2.0: https://github.com/conan-io/conan/issues/3620